README.md

```diff
@@ -71,7 +71,15 @@ mustache sample_workflow_view.json sample_workflow.json.mustache > sample_workfl
 ```
 
 You can find more about the command `mustache`
-[here](https://github.com/janl/mustache.js/), and below is an example of
+[here](https://github.com/janl/mustache.js/). 
+
+If `mustache` command is not found after install, run 
+```
+npm bin mustache
```

Comment:
`npm install -g mustache` is probably easier so that you don't need to add it to the PATH

Reply:
good point! updated.